keel-sql/src/main/resources/db/changelog/20200730-add-artifact-type-column.yml

```diff
@@ -0,0 +1,25 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-artifact-type-column
+      author: lpollo
+      changes:
+        - addColumn:
+            tableName: environment_artifact_queued_approval
+            columns:
+              - name: artifact_type
+                type: varchar(255)
```

Comment:
I think this should have:
`afterColumn: artifact_version` so it's not at the end of the table.

Also, do you ever query using artifact type? we might need an index on it if you do anything like "get all pending debians for this environment"

Reply:
oh i didn't know about `afterColumn` trick :) 
I'm not querying by type only, we query by type + version + artifact type+ environment uid from what I can tell.

Reply:
Sounds like it should be added into that index then, no?

Reply:
added.